docker-compose.yml

```diff
@@ -171,41 +223,47 @@ services:
     ports:
       - "3006:3006"
 
-  medicaid_gateway_helper:
-    build:
-      context: ../medicaid_gateway
-      dockerfile: ../ea_enterprise/site/medicaid_gateway_service/Dockerfile
-    image: medicaid_gateway:dev-0.1.0
-    volumes:
-      - ../medicaid_gateway:/medicaid_gateway
-      - mcg_rails_cache:/medicaid_gateway/tmp/cache
-      - mcg_bundle:/bundle
-      - mcg_node_modules:/medicaid_gateway/node_modules
-      - mcg_packs:/medicaid_gateway/public/packs
-      - ${PWD}/hotpatches/MG-stimulus.rb:/medicaid_gateway/config/initializers/stimulus_reflex.rb
+  # medicaid_gateway_helper:
```

Comment:
Is there a reason we're commenting this out?  How will we handle client toggling in MG?